packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -94,6 +83,22 @@ export function activate(context: vscode.ExtensionContext) {
     debugConfigProvider,
     checkpointsView
   );
+
+  await checkpointService.activateRequestService();
```

Comment:
I think you need error handling here. If there’s no org, this doesn’t look like it would exit cleanly and inform the user with what the error is and what they can do to resolve.